bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CredentialFactory.java

```diff
@@ -135,21 +135,6 @@ public static Credentials getCredentials(CredentialOptions options) throws IOExc
     }
   }
 
-  /**
-   * Initializes OAuth2 credential using preconfigured ServiceAccount settings on the local
-   * Google Compute Engine VM. See:
-   * <a href="https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances"
-   * >Creating and Enabling Service Accounts for Instances</a>.
-   *
-   * @return a {@link com.google.auth.Credentials} object.
-   * @throws java.io.IOException if any.
-   * @throws java.security.GeneralSecurityException if any.
-   */
-  public static Credentials getCredentialFromMetadataServiceAccount()
```

Comment:
Could this used publicly?

Reply:
Users should not use CredentialFactory directly.  If they need to replicate this logic, they can use `ComputeEngineCredentials` themselves.  This code was a copy from somewhere else that needed it, but for CBT, we don't need this.